drivers/storage/portworx/component/prometheus.go

```diff
@@ -511,6 +511,9 @@ func getPrometheusOperatorDeploymentSpec(
 				},
 				Spec: v1.PodSpec{
 					ServiceAccountName: PrometheusOperatorServiceAccountName,
+					NodeSelector: map[string]string{
```

Comment:
Can this be part of the NodeAffinity for all components, including the windows daemonset we are creating?